al_aws.js

```diff
@@ -33,25 +33,24 @@ var selfUpdate = function (callback) {
     });
 };
 
-var getMetricStatistics = function (params, statistics, callback) {
+var getMetricStatistics = function (params, callback) {
```

Comment:
as there is backward incompatible interface change, I owe you https://github.com/alertlogic/cwe-collector update as well ;)